### PR TITLE
Add Semantic Kernel for Java

### PR DESCRIPTION
**Pull request by brunoborges**

### CODE REVIEW

**Comment by weshaggard** [comment]
@brunoborges can you give me context on this PR?

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -1315,3 +1315,5 @@
 "azure-toolkit-cognitiveservices-lib","com.microsoft.azure","","0.46.0","Unknown Display Name","Unknown Service","NA","NA","NA","tool","false","","","","","","","","","","",""
 "azure-webapp-maven-plugin","com.microsoft.azure","2.13.0","","Webapp Maven Plugin","Plugin","NA","NA","NA","tool","false","","","","","","","","","","",""
 "webapp-maven-plugin","com.microsoft.azure","","0.1.1","Webapp Maven Plugin","Plugin","NA","NA","NA","tool","false","","","","","","","","","","",""
+"Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:1318]: Even if we add the package manually here we need to remove this line with the column headings here.

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -1315,3 +1315,5 @@
...
+"semantic-kernel-java","com.microsoft.semantic-kernel","1.1.5","","Semantic Kernel for Java","Semantic Kernel for Java","https://github.com/microsoft/semantic-kernel-java","https://learn.microsoft.com/semantic-kernel","tool","true","","","","community","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:1319]: How is this related to azure?

Comment (brunoborges) [reply _data/releases/latest/java-packages.csv:1319]: If this is not the right way to add the Semantic Kernel for Java SDK to the Java API browser, what is the right way?

**Comment by brunoborges** [comment]
> @brunoborges can you give me context on this PR?

I need to include the Semantic Kernel for Java SDK to the Java API browser

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:1319]: Our automation will generally only pull in "com.azure" or "com.microsoft.azure" packages in. Why should this one be added as part of the set of azure packages?

Comment (brunoborges) [reply _data/releases/latest/java-packages.csv:1319]: I have not found any other Java library in the API Browser that is not Azure.

What would be the ideal way to include a Java library to the API Browser if not through this?

**Comment by ronniegeraghty** [comment]
Adding this library here would put the library under the Azure SDK for Java section of the Java API browser. Since this library is not for a specific azure service and not a part of the Azure SDK for Java namespace or in the Azure SDK for Java repo, this is probably not the right section of the Java API Browser for this library.

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:1319]: I suspect you would need to work with the API Browser folks as I don't think this is the correct way to get added.

Comment (brunoborges) [reply _data/releases/latest/java-packages.csv:1319]: @weshaggard do you have a contact to share?

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:1319]: Unfortunately I do not. @ronniegeraghty or @JonathanGiles any idea who would be a good contact?

Comment (ronniegeraghty) [reply _data/releases/latest/java-packages.csv:1319]: I'll share a contact in a private message to @brunoborges
